mono/metadata/sre.c

```diff
@@ -3526,12 +3571,50 @@ remove_instantiations_of_and_ensure_contents (gpointer key,
 		return FALSE;
 }
 
+/**
+ * reflection_setup_internal_class:
+ * @tb: a TypeBuilder object
+ * @error: set on error
+ *
+ * Creates a MonoClass that represents the TypeBuilder.
+ * This is a trick that lets us simplify a lot of reflection code
+ * (and will allow us to support Build and Run assemblies easier).
+ *
+ * Returns TRUE on success. On failure, returns FALSE and sets @error.
+ */
+static gboolean
+reflection_setup_internal_class (MonoReflectionTypeBuilderHandle ref_tb, MonoError *error)
+{
+	MonoReflectionModuleBuilder *module = MONO_HANDLE_GETVAL (ref_tb, module);
```

Comment:
should be a handle for the module: `MonoReflectionModuleBuilderHandle module = MONO_HANDLE_NEW_GET (...)`